tests/queries/0_stateless/03174_statistics_countminsketch.sql

```diff
@@ -0,0 +1,78 @@
+-- Tags: no-fasttest
```

Comment:
This test is copypasta of 02864_statistics_uniq. We need to stop copypasting stuff, otherwise the tests will at some point crumble under the weight of their bloat.

Kindly consolidate this test with the existing test, and ideally with existing tests for tDigest-type statistics as well. Try to be as systematic as possible in the kind of operations tested, table and column names, and queries.